test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +54,33 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+TEST(ProcSysNetIpv4IpForward, Exists) {
+  auto fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/proc/sys/net/ipv4/ip_forward", O_RDWR));
+
+  fd.reset();
+}
+
+TEST(ProcSysNetIpv4IpForward, CanReadAndWrite) {
```

Comment:
Optional: I like to divide tests like this into two:
One for checking the default value.
One to check that the file is writable.

Example:
https://github.com/google/gvisor/blob/eefa817cfdb04ff07e7069396f21bd6ba2c89957/test/syscalls/linux/tcp_socket.cc#L493-L522

Reply:
I've added a test for checking the default value